mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/343 attack log format

Closes #343 

To test:
- Run Patches scenario and hit play
- When attack log starts flashing, open it and make sure text doesn't overflow past list item
